passes/techmap/extract.cc

```diff
@@ -427,6 +438,10 @@ struct ExtractPass : public Pass {
 		log("    -mine_max_fanout <num>\n");
 		log("        don't consider internal signals with more than <num> connections\n");
 		log("\n");
+		log("    -min_port_width <num>\n");
+		log("        match all subcircuits with port-sizes between needle port widths and this number\n");
```

Comment:
So "this number" is lower than the needle port widths?

Reply:
Yes, this number defines the lower bounds of the needle port width (it applies to all ports).  
Usually the port widths must match exactly, this exposes an existing feature of the algorithm.
So: hay_port_width ∈ [min_width; needle_port_width]

Reply:
Yes, I am asking due to the phrasing, which suggested to me `<num>` is an upper bound, not a lower bound. Not sure what we should change it to though.

Reply:
maybe something like:
```
match all subcircuits with port widths from <num> up to the needle port width given in the map.
```